algorithms/unit_tests/TestStdAlgorithmsMinMaxElementOps.cpp

```diff
@@ -202,7 +202,11 @@ void std_algo_min_max_test_verify(Kokkos::pair<IndexType, ValueType> goldPair,
                                   const ItType result,
                                   TestedViewType testedView) {
   // check that iterator is pointing to right element
+  // FIXME_OPENMPTARGET - With LLVM/13 we need PR 4456 to be merged. Else the
+  // following line fails duirng compile time.
+#ifndef KOKKOS_ENABLE_OPENMPTARGET
   EXPECT_EQ(result - KE::begin(testedView), goldPair.first);
+#endif
```

Comment:
```
/var/jenkins/workspace/Kokkos/algorithms/unit_tests/TestStdAlgorithmsMinMaxElementOps.cpp:203:50: error: unused parameter 'testedView' [-Werror,-Wunused-parameter]
                                  TestedViewType testedView) {

Reply:
Yes. Apparently I cannot avoid this with an #ifdef. This needs #4456 . That will fix this compile time issue.

Reply:
You can always do
```
#ifndef KOKKOS_ENABLE_OPENMPTARGET
  EXPECT_EQ(result - KE::begin(testedView), goldPair.first);
#else
  (void) testedView;
#endif
```

Reply:
I have tried that but looks like the CI still checks that line and fails at compile time. I was expecting the other PR to get merged before this and then we would not need the additional #ifdef.